src/coop/rchain/lang/formatter/RholangBlock.kt

```diff
@@ -0,0 +1,61 @@
+package coop.rchain.lang.formatter
```

Comment:
Please reformat code files to have indentation specified in `.editorconfig` with 2 spaces. Later modifications will be much cleaner.

Although on [KotlinLang](https://kotlinlang.org/docs/reference/coding-conventions.html#formatting) they say it's usually 4 spaces. What do you think, should we switch to 4 spaces? It's the similar question for Rholang.

Reply:
It is ok for 2 spaces when in KotlinLang. 
And for Rholang, I will create a new PR for add setting support for these preferred style.